Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.h

```diff
@@ -29,6 +29,7 @@ namespace O3DE::ProjectManager
     QT_FORWARD_DECLARE_CLASS(GemModel)
     QT_FORWARD_DECLARE_CLASS(GemSortFilterProxyModel)
     QT_FORWARD_DECLARE_CLASS(DownloadController)
+    QT_FORWARD_DECLARE_CLASS(CreateAGemScreen)
```

Comment:
```suggestion
```
this forward declare should not be necessary

Reply:
Removed.